### ISSUE TITLE
TensorFlow throws an exception when loading a model that was normalized like this.

### ISSUE THREAD

**Issue by rafak360** [tensorflow/tensorflow#51911]
See my stackoverflow question that covers up everything:
https://stackoverflow.com/questions/69112466/why-tensorflow-throws-this-exception-when-loading-a-model-that-was-normalized-li

**Comment by rafak360** [comment]
In addition to that, using that same code example from stackoverflow will notice a significant loss of accuracy after the model is loaded properly (with input_dim specified in Normalization()), however that does NOT happen in tensorflow-gpu 2.5.0 which means tensorflow-gpu 2.6.0 has more bugs than expected.

**Comment by tilakrayal** [comment]
@rafak360 ,
I was able to execute the mentioned [SO code](https://stackoverflow.com/questions/69112466/why-tensorflow-throws-this-exception-when-loading-a-model-that-was-normalized-li) in tf v2.6 without any issue.Please find the gist of it [here](https://colab.research.google.com/gist/tilakrayal/845d0574f6411100fc00255bfcfc0b62/un51911.ipynb).Also i can see your  tf v2.6 is installed with Cuda 11.4.2 which is incompatible.Please check the link for tested build [configurations](https://www.tensorflow.org/install/source#gpu) and try to install with compatible version.Thanks!

**Comment by rafak360** [comment]
> @rafak360 ,
> I was able to execute the mentioned [SO code](https://stackoverflow.com/questions/69112466/why-tensorflow-throws-this-exception-when-loading-a-model-that-was-normalized-li) in tf v2.6 without any issue.Please find the gist of it [here](https://colab.research.google.com/gist/tilakrayal/845d0574f6411100fc00255bfcfc0b62/un51911.ipynb).Also i can see your  tf v2.6 is installed with Cuda 11.4.2 which is incompatible.Please check the link for tested build [configurations](https://www.tensorflow.org/install/source#gpu) and try to install with compatible version.Thanks!

You forgot to edit that code and uncomment the part  #model = tf.keras.models.load_model('AI/test.h5') so that is where the problem begins. I tried your  gist and achieved same exception. You might wanna change the path dir from AI/test.h5 to just test.h5 since gist seems not to accept creating a sub folder. So run the code for the first time just as is, then for the second time uncomment that part and load the model then boom exception is thrown. I switched to the compatible versions as you mentioned and still the bug, note that there are 2 bugs I'm reporting, the second bug is the loss of accuracy when loading a model.

**Comment by tilakrayal** [comment]
@rafak360 ,
Please post this issue on [keras-team/keras](https://github.com/keras-team/keras/issues) repo.
To know more refer to:
[https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999](https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999)

**Comment by rafak360** [comment]
> @rafak360 ,
> Please post this issue on [keras-team/keras](https://github.com/keras-team/keras/issues) repo.
> To know more refer to:
> https://discuss.tensorflow.org/t/keras-project-moved-to-new-repository-in-https-github-com-keras-team-keras/1999

Done: https://github.com/keras-team/keras/issues/15348

**Comment by tilakrayal** [comment]
@rafak360 ,
Please feel free to move this issue to closed status as it has been tracking in Keras repo.Thanks!
